kotti/resources.py

```diff
@@ -495,6 +495,16 @@ def _not_root(context, request):
     return not context is get_root()
 
 
+default_actions = LinkParent(title=_(u'Actions'), children=[
+    Link('copy', title=_(u'Copy')),
+    Link('cut', title=_(u'Cut'), predicate=_not_root),
+    Link('paste', title=_(u'Paste'), predicate=get_paste_items),
+    Link('rename', title=_(u'Rename'), predicate=_not_root),
+    Link('delete', title=_(u'Delete'), predicate=_not_root),
+    LinkRenderer('default-view-selector'),
+])
+
```

Comment:
Hmm… wouldn't it be better to have ``default_actions = […]`` (list of links) and ``LinkParent(title=_(u'Actions'), children=default_actions)`` in line 507?

Would be both easier to read and modify.